templates/repo/issue/new_form.tmpl

```diff
@@ -136,6 +136,64 @@
 				</div>
 			</div>
 
+			{{if .IsProjectsEnabled}}
+			<div class="ui divider"></div>
+
+			<input id="project_id" name="project_id" type="hidden" value="{{.project_id}}">
+			<div class="ui {{if not .HasIssuesOrPullsWritePermission}}disabled{{end}} floating jump select-project dropdown">
+			  <span class="text">
+			    <strong>{{.i18n.Tr "repo.issues.new.projects"}}</strong>
+			    {{if .HasIssuesOrPullsWritePermission}}
+			      {{svg "octicon-gear" 16}}
+			    {{end}}
+			  </span>
+			  <div class="menu">
+			    <div class="header" style="text-transform: none;font-size:16px;">{{.i18n.Tr "repo.issues.new.add_project_title"}}</div>
+			    {{if or .OpenProjects .ClosedProjects}}
+			    <div class="ui icon search input">
+			      <i class="search icon"></i>
```

Comment:
Very minor but there's 6 such FontAwesome icons in use in this PR and I'd prefer those to be replaced with octicons (I want to eventually eliminate FA).